packages/components/src/components/conditional-wrapper/condtional-wrapper.jsx

```diff
@@ -0,0 +1,3 @@
+const ConditionalWrapper = ({ children, condition, wrap }) => (condition ? wrap(children) : children);
```

Comment:
Do we need this to have only one condition? Other component wrappers may require other conditions or no condition at all. 
How about we create a wrapper in p2p package that is specific to the component?